packages/clay-language-picker/src/index.tsx

```diff
@@ -0,0 +1,285 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {Option, Picker} from '@clayui/core';
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import ClayLayout from '@clayui/layout';
+import {InternalDispatch, sub} from '@clayui/shared';
+import classNames from 'classnames';
+import React from 'react';
+
+type DisplayType = 'info' | 'secondary' | 'success' | 'warning';
+
+type Item = {
+	displayName?: string;
```

Comment:
Just a suggestion of preference, maybe `name` or `label` for the name of this prop. What do you think?

Reply:
`name` sounds good